cypress/e2e/webTables.cy.js

```diff
@@ -1,5 +1,193 @@
+/// <reference types='cypress' />
+
+import { generateUser } from '../support/generateUser';
+
 describe('Web Tables page', () => {
-  it('', () => {
+  beforeEach(() => {
+    cy.visit('/');
+  });
+
+  it('should add new worker', () => {
+    const {
+      firstName,
+      lastName,
+      email,
+      age,
+      salary,
+      department
+    } = generateUser();
+
+    cy.get('[title="Delete"]').should('have.length', '3');
+    cy.get('#addNewRecordButton').click();
+    cy.get('#firstName').type(firstName);
+    cy.get('#lastName').type(lastName);
+    cy.get('#userEmail').type(email);
+    cy.get('#age').type(age);
+    cy.get('#salary').type(salary);
+    cy.get('#department').type(`${department}{Enter}`);
+    cy.get('[title="Delete"]').should('have.length', '4');
+  });
+
+  it('should delete all workers', () => {
+    if (cy.get('[title=Delete]').should('have.length.at.least', 1)) {
+      cy.get('[title=Delete]')
+        .then(rows => {
+          for (let i = 0; i < rows.length; i++) {
+            cy.get('[title=Delete]').first().click();
+          }
+        });
+
+      cy.get('.web-tables-wrapper').should('contain.text', 'No rows found');
```

Comment:
The test case 'should delete all workers' doesn't assert that all workers have been deleted. It only asserts that 'No rows found' text is present. This is not a complete validation